indicators/indicators.c

```diff
@@ -29,6 +29,10 @@ bool draw_indicators(uint8_t led_min, uint8_t led_max) {
     uint8_t mods  = get_mods();
     uint8_t layer = get_highest_layer(layer_state);
 
+#ifndef NO_ACTION_ONESHOT
+    mods |= get_oneshot_mods();
+#endif // NO_ACTION_ONESHOT
```

Comment:
I'd rather not have the comment, since this is a one-liner and we can easily see its opening `#if`

Reply:
Fair enough, just force of habbit for me.